src/main/java/com/gw2auth/oauth2/server/service/gw2account/verification/Gw2AccountVerificationServiceImpl.java

```diff
@@ -328,4 +334,11 @@ private void recordMetric(Gw2AccountVerificationChallengePendingEntity entity, b
     private <S> Map<String, Object> buildMessage(VerificationChallenge<S> challenge, String rawState) throws IOException {
         return challenge.buildMessage(challenge.readState(rawState));
     }
+
+    private static class NoRollbackServiceException extends Gw2AccountVerificationServiceException {
```

Comment:
Not a fan of the name, to be honest. What do yu think of `[Expected|Handled]Gw2AccountVerificationServiceException`?

Reply:
`HandledGw2AccountVerificationServiceException` sounds good 👍